internal/riverinternaltest/riverdrivertest/riverdrivertest.go

```diff
@@ -216,41 +414,49 @@ func Exercise[TTx any](ctx context.Context, t *testing.T,
 		exec, _ := setup(ctx, t)
 
 		exists, err := exec.ColumnExists(ctx, &riverdriver.ColumnExistsParams{
-			Column: "id",
-			Table:  "river_job",
+			Column: "line",
+			Table:  "river_migration",
 		})
 		require.NoError(t, err)
 		require.True(t, exists)
 
 		exists, err = exec.ColumnExists(ctx, &riverdriver.ColumnExistsParams{
 			Column: "does_not_exist",
-			Table:  "river_job",
+			Table:  "river_migration",
 		})
 		require.NoError(t, err)
 		require.False(t, exists)
 
 		exists, err = exec.ColumnExists(ctx, &riverdriver.ColumnExistsParams{
-			Column: "id",
+			Column: "line",
 			Table:  "does_not_exist",
 		})
 		require.NoError(t, err)
 		require.False(t, exists)
 
-		// Will be rolled back by the test transaction.
-		_, err = exec.Exec(ctx, "CREATE SCHEMA another_schema_123")
-		require.NoError(t, err)
-
-		_, err = exec.Exec(ctx, "SET search_path = another_schema_123")
-		require.NoError(t, err)
+		// A different schema on main, but before the `line` column was added to
+		// migrations.
+		driver2, schemaVersion2 := driverWithSchema(ctx, t, &riverdbtest.TestSchemaOpts{
+			LineTargetVersions: map[string]int{
+				riverdriver.MigrationLineMain: 2,
+			},
+		})
 
-		// Table with the same name as the main schema, but without the same
-		// columns.
-		_, err = exec.Exec(ctx, "CREATE TABLE river_job (another_id bigint)")
-		require.NoError(t, err)
+		// In SQLite, the schema needs to be attached. No-op for Postgres.
+		//
+		// Notably, the new schema is not detached. I initially tried to do so,
+		// but it can't be detached while the test transaction is running (you
+		// get an error like "SQL logic error: database
+		// river_test_2025_04_30t12_26_17_schema_03 is locked (1)"), and rolling
+		// it back early means that a detach can no longer be run on it. It
+		// doesn't matter much if an extra schema is attached until the tests
+		// finish running.
+		// require.NoError(t, exec.SchemaAttach(ctx, &riverdriver.SchemaAttachParams{Schema: schemaVersion2}))
```

Comment:
making sure you meant to leave this in here

Reply:
Ah nope. That was part of a previous attempt in getting things to work and no longer needed. I also removed one other similar block in this file that was also commented out.